### ISSUE TITLE
Handling SDK updates

### ISSUE THREAD

**Issue by ErikEJ** [microsoft/DacFx#494]
Currently there is no Nuget mechanism to check for SDK updates.

Any thoughts about how to handle this with Microsoft.Build.Sql ?

(We have the same issue with MsBuild.SDK.Sqlproj)

**Comment by zijchen** [comment]
I agree it's not ideal having to update the SDK version manually every time. Unfortunately it doesn't seem that our friends at .NET have plans on supporting this: https://github.com/dotnet/msbuild/issues/3099

We may consider this as an IDE feature.

**Comment by ErikEJ** [comment]
We added this check to MsBuild.Sdk:sqlProj: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/pull/643

**Comment by ErikEJ** [comment]
@zijchen Could this be closed as fixed now?
